src/Control/Monad/Ghc.hs

```diff
@@ -25,12 +27,22 @@ instance (Functor m, Monad m) => Applicative (GhcT m) where
   pure  = return
   (<*>) = ap
 
+-- adapted from https://github.com/ghc/ghc/blob/59fa7b32b018a91f81773ca676251a0b2761ef56/compiler/main/GHC.hs#L454-L463
```

Comment:
Would it be possible to use a tag for the link here? Would make the link a bit more human-friendly and shorter.

We then do depend on then not modifying tags, but I doubt they will. And if they do, it's unlikely that the line numbers will be wrong anyway.

Reply:
done